composer.json

```diff
@@ -24,14 +25,20 @@
         "typo3/coding-standards": "^0.7.1"
     },
     "replace": {
-        "t3docs/php-domain": "self.version",
         "t3docs/typo3-docs-theme": "self.version",
         "t3docs/typo3-guides-extension": "self.version"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/brotkrueml/twig-codehighlight"
```

Comment:
Could you add your package on packagist, so that a distinct repository wouldn't be needed?

Reply:
Yup, as far as I have convinced myself it should be ;-)

Reply:
I give you a magic +1 Stick of Conviction! :-D